packages/bot-web-ui/src/stores/dashboard-store.ts

```diff
@@ -28,16 +35,42 @@ export default class DashboardStore implements IDashboardStore {
     @observable faq_search_value = null || '';
     @observable dialog_options = {};
     @observable is_dialog_open = false;
+    @observable getFileArray = [];
+    @observable has_file_loaded = false;
     @observable is_info_panel_visible = true;
 
     @action.bound
-    onCloseDialog(): void {
+    setFileLoaded = (has_file_loaded: boolean): void => {
+        this.has_file_loaded = has_file_loaded;
+        clearInjectionDiv();
+    };
+
+    @action.bound
+    onCloseDialog = (): void => {
         this.is_dialog_open = false;
-    }
+    };
+
     @action.bound
-    setActiveTab(active_tab: number): void {
+    setActiveTab = (active_tab: number): void => {
         this.active_tab = active_tab;
+    };
+
+    @action.bound
+    showVideoDialog(param: { [key: string]: string | React.ReactNode }): void {
+        const { type, component, url } = param;
+        if (type === 'DBotVideo') {
+            this.dialog_options = {
```

Comment:
what is the difference between this `if` and the following `else`? 
tehy both have the same code

Reply:
done ✅